<launch>

    <node   pkg="bebop2_ground_control"
            type="ground_control_node"
            name="ground_control_node_sphinx"
            respawn="false"
            launch-prefix=""
    		required="true">

            <param name="node_rate" value="20"/>

            <remap from="/bebop_auto/cmd_vel" to="/bebop2_auto/cmd_vel"/>

            <remap from="/bebop/takeoff" to="/bebop2/takeoff"/>
            <remap from="/bebop/land" to="/bebop2/land"/>
            <remap from="/bebop/cmd_vel" to="/bebop2/cmd_vel"/>
            <remap from="/bebop/cmd_vel_stamped" to="/bebop2/cmd_vel_stamped"/>

    </node>

</launch>
